Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -278,6 +278,7 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
         {
             // No need to select the folder ourselves, callback from Breadcrumbs will take care of that
             m_ui->m_pathBreadCrumbs->pushPath(QString::fromUtf8(folderEntry->GetVisiblePath().c_str()));
+            m_ui->m_pathBreadCrumbs->setFullPath(QString::fromUtf8(folderEntry->GetFullPath().c_str()));
```

Comment:
do we need both of these calls? Is it possible to either make it so that pushPath knows how to build the fullPath and sets it itself, or maybe have setFullPath know to push the end string? I'm just suspicious that everywhere that pushPath is called, setFullPath is called too, so it seems like these might be able to be combined.

Reply:
Done! I've combined the methods, so pushFullPath calls pushPath and sets itself.